cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -895,7 +895,7 @@ export class ArmatureDisplay extends UIRenderer {
         if (!this._playing) {
             if (frameCache.isInvalid()) {
                 frameCache.updateToFrame();
-                this._curFrame = frames[frames.length - 1];
+                this._curFrame = frames[frames.length - 1]!;
```

Comment:
Why need to add `!` now?

Reply:
because we know it is not undefined, we don't need the TSC to tell us this could be error, for example
```ts
for (let i = 0 ; i < aList.lenth; ++i) {
   aList[i]!  // I know it's not undefined, so ts don't need to emit error
}
```

Reply:
Got it. It seems SafeArray brings some inconvenient.

Reply:
yes, but it's type safer